README.md

```diff
@@ -1,6 +1,6 @@
 # Kompose (Kubernetes + Compose)
 
-[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
+[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]  [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes/kompose)](https://goreportcard.com/report/github.com/kubernetes/kompose)
```

Comment:
Hi, please add the URL to the bottom of the page, you'll see where the other URL's are.